content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -5304,6 +5304,32 @@ Informs about messages sent via the `multicast` method or by the system.
 | `content`   | Yes      |       |
 | `type`      | No       |       |
 
+### `chat_unfollowed`
+Informs that chat has been unfollowed.  Useful in multiple connections scenarios, where one connection needs to know that another one unfollowed chat.
+
+<CodeResponse title={'Sample push payload'}>
+
+```json
+{
+  "chat_id": "PJ0MRSHTDG"
+}
+```
+
+</CodeResponse>
+
+#### Specifics
+
+|                        |                   |
+| ---------------------- | ----------------- |
+| **Action**             | `chat_unfollowed` |
+| **Webhook equivalent** | -                 |
+
+#### Push payload
```

Comment:
I don't think this is needed. There is no magic to explain about `chat_id` and it is hard to `require` fields in a push.